frontend/src/globals/unity-util.ts

```diff
@@ -25,6 +25,8 @@ import { ExternalWebRequestHandler } from './unity-externalwebrequesthandler';
 declare let SendMessage;
 declare let createUnityInstance;
 
+type DrawingImageSource = ImageBitmap | ImageData | HTMLImageElement | HTMLCanvasElement | OffscreenCanvas;
```

Comment:
This is only a suggestion, but I think that having something like this would look nicer. I dont know if a box per se, but something that acompasses the whole bounding box.

<img width="405" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/806b9ff9-0336-4823-912d-adbe89ca2415">